layouts/index.html

```diff
@@ -16,36 +17,38 @@
     <link rel="stylesheet" href="{{ $styles.Permalink }}" media="screen">
   {{ end }}
 </head>
+
 <body>
 
   <div class="container">
-  	<img class="mascot" src="yegwit-owl-small-square.png" alt="YEGWIT Owl Mascot">
+    <img class="mascot" src="yegwit-owl-small-square.png" alt="YEGWIT Owl Mascot">
 
-  	<div>
-  		<header>
-  			<h1>YEGWIT</h1>
-  			<p>Edmonton Women in Tech</p>
-  		</header>
-  		<nav>
-  			<a href="https://twitter.com/yegwit"><i class="fab fa-twitter"></i></a>
-  			<a href="https://www.instagram.com/yegwit/"><i class="fab fa-instagram"></i></a>
-  			<a href="https://github.com/yegwit"><i class="fab fa-github"></i></a>
-  			<a href="https://www.eventbrite.com/o/yeg-women-in-tech-26752827013"><i class="far fa-calendar-alt"></i></a>
+    <div>
+      <header>
+        <h1>YEGWIT</h1>
+        <p>Edmonton Women in Tech</p>
+      </header>
+      <nav>
+        <a href="https://twitter.com/yegwit"><i class="fab fa-twitter"></i></a>
+        <a href="https://www.instagram.com/yegwit/"><i class="fab fa-instagram"></i></a>
+        <a href="https://github.com/yegwit"><i class="fab fa-github"></i></a>
+        <a href="https://www.eventbrite.com/o/yeg-women-in-tech-26752827013"><i class="far fa-calendar-alt"></i></a>
         <a href="email-sign-up.html"><i class="far fa-envelope"></i></a>
-  		</nav>
-  	</div>
+      </nav>
+    </div>
 
   </div>
   <main>
     <p>YEGWIT aims to build a strong and safe group for women and allies in tech
-      with the goal of fostering engagement, providing mentorship, and
-      creating community. We want to establish a group that allows individuals
-      to share ideas, concerns, thoughts and actions, to create an
-      equitable environment within the industry.</p>
-    <p>YEGWIT: Foster, Validate, Empower.</p>
+        with the goal of fostering engagement, providing mentorship, and
+        creating community. We want to establish a group that allows individuals
```

Comment:
Instead of "we want to", I'd say "We are establishing..." or something like that.